test/test_sam_annotator/test_widgets.py

```diff
@@ -0,0 +1,31 @@
+import json
+import os
+
+import zarr
+
+from micro_sam.sam_annotator._widgets import embedding_widget, Model
+from micro_sam.util import _compute_data_signature
+
+
+# make_napari_viewer is a pytest fixture that returns a napari viewer object
+# you don't need to import it, as long as napari is installed
+# in your testing environment.
+# tmp_path is a regular pytest fixture.
+def test_embedding_widget(make_napari_viewer, tmp_path):
```

Comment:
If possible we should also check that the `IMAGE_EMBEDDINGS` variable is updated here.